packages/ssi-service/package.json

```diff
@@ -31,7 +31,7 @@
     "css-loader": "3.5.3",
     "dotenv-webpack": "1.7.0",
     "eslint-plugin-jest": "23.11.0",
-    "jest": "25.5.4",
+    "jest": "26.0.1",
     "jest-environment-jsdom-sixteen": "1.0.3",
```

Comment:
Need to remove the `jest-environment-jsdom-sixteen` package as that was being used to enable jsdom v16 in the earlier versions of jest. It's not needed now that jest v26 has support of jsdom v16 in-built.